pkg/cli/mirror/operator.go

```diff
@@ -191,6 +191,9 @@ func (o *OperatorOptions) renderDCFull(ctx context.Context, reg *containerdregis
 			IncludeAdditively: includeAdditively,
 			SkipDependencies:  ctlg.SkipDependencies,
 		}.Run(ctx)
+		if err != nil {
+			return nil, err
+		}
 	}
```

Comment:
So the error is always caught for both branches
```suggestion
	}
	if err != nil {
		return nil, err
	}
```

Reply:
Oh I see what the issue is. Disregard this.